### ISSUE TITLE
[good first issue] Add your favorite pokemon

### ISSUE THREAD

**Issue by KarlHeitmann** [Strike2-ux/PokeApi-Node-and-Express#2]
The constant `allPokemon` currently has only 5 pokemons. Add your favorite pokemon in the array, so we can increment the database of pokemons

**Comment by Strike2-ux** [comment]
If you can see in the new update, it is update, please check

**Comment by RDiMag** [comment]
Is there room for another Pokemon? I'd like to contribute.

**Comment by Strike2-ux** [comment]
Yes you can @RDiMag , any contribution it’s welcome, the objective is to understand what t’s behind the backend, the thing is that already exist a Backend, https://pokeapi.co/ so instead of writing every single Pokémon we can use syntaxis, logical commands, libraries or a frontend framework or microframework (React like example), with libraries to use this backend and transformed like something like this https://emonsalvespokedex.netlify.app/ . Have a nice day!

**Comment by RDiMag** [comment]
I understand your comment to mean that you are building an API that can be used by others to create apps with React or other frameworks?

To contribute, would I create a PR with the Pokemon stats? Or are you looking for ways to import data from PokeApi.co? This is my first venture into opensource so I may be misunderstanding.

**Comment by Strike2-ux** [comment]
Don´t worry @RDiMag , feel comfortable, do whatever that you want, any contribution will be appreciated, but remember show us that you know what are you doing.

**Comment by Davidbacke** [comment]
Do you want us to simply add Pokémon to the "allPokemon" constant like this:

id:"422",
name: "shellos",
type0:"water",
url: "https://pokeapi.co/api/v2/pokemon/422/",
image_url: "https://assets.pokemon.com/assets/cms2/img/pokedex/full/422.png"

In the index.js file?

**Comment by Strike2-ux** [comment]
Of course David, think about that https://pokeapi.co/ is doing this with all the Lore of Pokemon, index.js for explain a simple API, the good practice of the frontend developers are to fetch all this petitions of a backend. Welcome @Davidbacke

**Comment by LeannonS** [comment]
hello everything is fine?
could i be adding the desired amount of pokemons to the project? If this is not a problem, I would like to add all remaining pokemons from the first generation.

**Comment by Strike2-ux** [comment]
Thank you @LeannonS  , all contributions will be gladly accepted, put all the Pokemon’s and lore that you can, it will be check in the review.
